### ISSUE TITLE
Use exif data to sort images

### ISSUE THREAD

**Issue by ghost** [IacobIonut01/Gallery#107]
Currently, it seems that Gallery doesn't use the exif data from photos to sort them, but rather uses the file timestamp. 

It would be nice if there was an option to use exif data to sort photos.

**Comment by IacobIonut01** [comment]
I'll be adding multiple sorting options for the media as well (not only for the albums - which is already available).

Including the DATE_TAKEN - the value that holds the exif timestamp.


Thanks for the request

**Comment by laraib07** [comment]
@IacobIonut01  Can you also add option to choose how to arrange photos, i.e whether to arrange date-wise or month-wise.

**Comment by IacobIonut01** [comment]
> @IacobIonut01  Can you also add option to choose how to arrange photos, i.e whether to arrange date-wise or month-wise. 

What do you mean by date-wise

**Comment by IacobIonut01** [comment]
For example showing photos in a single section per month? Instead of big Month header and per date small headers

**Comment by laraib07** [comment]
@IacobIonut01  Yes, date-wise is current behaviour. There is a big Month Header with multiple section for each date. It would be appreciated if you can also add option for showing a single section per month.

**Comment by IacobIonut01** [comment]
> @IacobIonut01  Yes, date-wise is current behaviour. There is a big Month Header with multiple section for each date. It would be appreciated if you can also add option for showing a single section per month.

Sure, I'll add the option

**Comment by IacobIonut01** [comment]
> Currently, it seems that Gallery doesn't use the exif data from photos to sort them, but rather uses the file timestamp.
> 
> It would be nice if there was an option to use exif data to sort photos.

The feature is still WIP, be aware that the exif timestamp will be available only for the Camera-taken Media (or if exif data_taken attribute exists). Other photos & videos will be sorted normally by the date_modified attribute. Might result in confusing from an UX level, which will take a bit more to properly implement it so the user will be able to control this kind of sorting and know when this sort happened

**Comment by k2helix** [comment]
Maybe simply add a sort option for Date taken (exif) and one for last modified (default). Apps like Google Photos or Simple Gallery Pro (open source but recently bought) have this feature.
